internal/evaluator/criteria.go

```diff
@@ -86,33 +122,8 @@ func computeIncludeExclude(src ecc.Source, p ConfigProvider) (*Criteria, *Criter
 
 	vc := src.VolatileConfig
 	if vc != nil {
-		at := p.EffectiveTime()
```

Comment:
This feels like a good change but unrelated? Maybe put it in a different commit?

Reply:
Yes, unrelated. I split it off. https://github.com/enterprise-contract/ec-cli/pull/2347/commits/488dba5fdb30f8ac272976411de24be0f3c80c55